docs/asciidoc/metrics.adoc

```diff
@@ -88,14 +88,17 @@ Just adding these two operators will expose a whole bunch of useful metrics!
 
 | [name].onNext.delay | Timer | Measures delays between onNext signals (or between onSubscribe and first onNext)
 
-| [name].flow.duration | Timer | Times the duration elapsed between a subscription and the termination or cancellation of the sequence. A status tag is added to specify what event caused the timer to end (`onComplete`, `onError`, `cancel`).
+| [name].flow.duration | Timer | Times the duration elapsed between a subscription and the termination or cancellation of the sequence. A status tag is added to specify what event caused the timer to end (`onComplete`, `onCompleteEmpty`, `onError`, `cancel`).
```

Comment:
actually, this documentation is wrong. the `status` values are `completed`, `cancelled` `completedEmpty` and `error` 😢 for the 3 old possible values, this was already wrong in 3.2 documentation though...